src/source-pcap-file-helper.h

```diff
@@ -32,6 +32,7 @@ typedef struct PcapFileGlobalVars_ {
     ChecksumValidationMode conf_checksum_mode;
     ChecksumValidationMode checksum_mode;
     SC_ATOMIC_DECLARE(unsigned int, invalid_checksums);
+    int read_buffer_size;
```

Comment:
signed integer ?

Reply:
Initially wanted -1 to mean unused. Will make it unsigned.